src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -625,22 +625,21 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
 
       def getInactiveBakersWithTaggedAccounts(
           taggedAccounts: List[BlockTagged[Map[AccountId, Account]]]
-      ): Future[List[(BlockTagged[Map[AccountId, Account]], List[Tables.AccountsHistoryRow])]] =
+      ): Future[List[(BlockTagged[Map[AccountId, Account]], List[Tables.AccountsRow])]] =
         Future.traverse(taggedAccounts) { blockTaggedAccounts =>
           if (blockTaggedAccounts.blockLevel % lorreConf.blockRightsFetching.cycleSize == 1) {
             tezosNodeOperator.fetchActiveDelegates(taggedAccounts.map(x => (x.blockLevel, x.blockHash))).flatMap {
               activeBakers =>
                 db.run {
                   TezosDb
-                    .getInactiveBakersFromAccountsHistoryByBlock(
-                      blockTaggedAccounts.blockLevel,
+                    .getInactiveBakersFromAccounts(
```

Comment:
is the existing `getInactiveBakersFrom AccountsHistoryByBlock` used elsewhere still?
Otherwise we might get rid of that.

Reply:
no, I removed it